ui/shared/AssetSelector.qml

```diff
@@ -0,0 +1,166 @@
+                anchors.verticalCenter: parent.verticalCenter
+                width: 40
+                height: 40
+                sourceSize.height: height
+                sourceSize.width: width
+                fillMode: Image.PreserveAspectFit
+                source: "../app/img/tokens/" + symbol.toUpperCase() + ".png"
+            }
+            Column {
+                anchors.left: iconImg.right
+                anchors.leftMargin: 12
+                anchors.verticalCenter: parent.verticalCenter
+
+                StyledText {
+                    text: symbol.toUpperCase()
+                    font.pixelSize: 15
+                    height: 22
+                }
+
+                StyledText {
+                    text: name
+                    color: Style.current.grey
+                    font.pixelSize: 15
+                    height: 22
+                }
+            }
+            Column {
+                anchors.right: parent.right
+                anchors.rightMargin: Style.current.padding
+                anchors.verticalCenter: parent.verticalCenter
+                StyledText {
+                    font.pixelSize: 15
+                    height: 22
+                    text: parseFloat(value).toFixed(4) + " " + symbol
+                }
+                StyledText {
+                    font.pixelSize: 15
+                    anchors.right: parent.right
+                    height: 22
+                    text: fiatValue.toUpperCase()
+                    color: Style.current.grey
+                }
+            }
+            background: Rectangle {
+                color: itemContainer.bgColor
+                radius: Style.current.radius
+
+                // cover bottom left/right corners with square corners
+                Rectangle {
+                    visible: !isLastItem
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.bottom: parent.bottom
+                    height: parent.radius
+                    color: parent.color
+                }
+
+                // cover top left/right corners with square corners
+                Rectangle {
+                    visible: !isFirstItem
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.top: parent.top
+                    height: parent.radius
+                    color: parent.color
+                }
+            }
+            MouseArea {
+                cursorShape: Qt.PointingHandCursor
+                anchors.fill: itemContainer
+                hoverEnabled: true
+                onEntered: {
+                    itemContainer.bgColor = Style.current.grey // change to lightGrey after https://github.com/status-im/nim-status-client/pull/698 is merged
+                }
+                onExited: {
+                    itemContainer.bgColor = Style.current.white
```

Comment:
Assumption on my part:
```suggestion
                    itemContainer.bgColor = Style.current.background
```

Reply:
Agreed, I will address this in an updated https://github.com/status-im/nim-status-client/pull/698